nf_core/pipeline-template/.github/workflows/release-announcements.yml

```
@@ -20,6 +25,8 @@ jobs:
 
             Please see the changelog: ${{ github.event.release.html_url }}
 
+            ${{ steps.get_topics.outputs.GITHUB_OUTPUT }} #nfcore #openscience
```

```suggestion
            ${{ steps.get_topics.outputs.GITHUB_OUTPUT }} #nfcore #openscience #nextflow #bioinformatics
```

Based on the hasthtags I find are more busy